packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -255,7 +255,7 @@ const releaseApi = createApi({
             method: 'DELETE',
           };
         },
-        invalidatesTags: (result, error, arg) => [{ type: 'Release', id: arg.id }],
+        invalidatesTags: () => [{ type: 'Release', id: 'LIST' }],
```

Comment:
I thought that was the issue 🙃 , pretty sure you could just return an array directly instead of a function that returns an array.

Reply:
you're right! forgot about that